docs/src/understand/single-sign-on/trouble-shooting.md

```diff
@@ -1,88 +1,174 @@
 (trouble-shooting-faq)=
 
+```{contents}
+:depth: 2
+```
+
 # Trouble shooting & FAQ
 
 ## Reporting a problem with user provisioning or SSO authentication
 
 In order for us to analyse and understand your problem, we need at least the following information up-front:
 
 - Have you followed the following instructions?
-  : - {ref}`FAQ <trouble-shooting-faq>` (This document)
+    - {ref}`FAQ <trouble-shooting-faq>` (This document)
     - [Howtos](https://docs.wire.com/how-to/single-sign-on/index.html) for supported vendors
     - [General documentation on the setup flow](https://support.wire.com/hc/en-us/articles/360001285718-Set-up-SSO-externally)
-- Vendor information (octa, azure, centrica, other (which one)?)
-- Team ID (looks like eg. `2e9a9c9c-6f83-11eb-a118-3342c6f16f4e`, can be found in team settings)
+- Which vendor (and product if /a) are you using (octa, azure, centrica, other (which one)?)
+- Team ID (looks like eg. `2e9a9c9c-6f83-11eb-a118-3342c6f16f4e`, can be found in the team management app)
+- User ID of the account that has the problem (alternatively: handle, email address)
 - What do you expect to happen?
-  : - eg.: "I enter login code, authenticate successfully against IdP, get redirected, and see the wire landing page."
+    - eg.: "I enter login code, authenticate successfully against IdP, get redirected, and see the wire landing page."
```

Comment:
```suggestion
    - e.g.: "I enter login code, authenticate successfully against IdP, get redirected, and see the wire landing page."
```

Reply:
i don't like this, it's correct but awkward...  but ok!